typescript/packages/subsurface-viewer/src/layers/wells/utils/spline.test.ts

```diff
@@ -44,4 +82,27 @@ describe("remove duplicates", () => {
             removeConsecutiveDuplicates(coordsEmpty, mdsEmpty)[1]
         ).toStrictEqual(mdsEmpty);
     });
+
+    // Test splineRefine functions
+    it("should not refine if given invalid input", () => {
+        expect(
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
+            splineRefine(testWell as unknown as FeatureCollection, 0).features[0].geometry.geometries[1].coordinates.length // eslint-disable-line
```

Comment:
Could you define the `testWell` const as `FeatureCollection` instead of casting it here?